assets/js/components/ClusterDetails/ChecksResults.jsx

```diff
@@ -54,6 +56,8 @@ const getResultIcon = (executionState, health) => {
 };
 
 export const ChecksResults = () => {
+  const [modalOpen, setModalOpen] = useState(false);
+  const [selectedCheck, setSelectedCheck] = useState(false);
```

Comment:
Maybe better to default to an empty string or null? This value is a string after all, and not a boolean
`useState("")`